examples/medplum-postman-examples/Medplum Environment.postman_environment.json

```diff
@@ -0,0 +1,33 @@
+{
+	"id": "cbb624b4-c4a2-4914-bcf4-5ccfa1dca857",
+	"name": "Medplum Environment",
```

Comment:
Could we rename to `Medplum` and rename the file to `medplum.postman_environment.json`?